<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Modify</class>
 <widget class="QDialog" name="Modify">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="updateButtonBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>540</y>
     <width>621</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="updateNom">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>621</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Fiche d'Edition</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNom">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Titre</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="updateEdition">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>621</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelEdition">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Edition</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="updateSocPri">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>621</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelSocPri">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>SocPri</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>updateButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>Modify</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>updateButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>Modify</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
